content/internships/applicant-guide/contents.lr

```diff
@@ -72,6 +72,8 @@ All the CC mentors are on [Slack](/community). **Please use public channels on S
 
 Remember that the mentors are busy – we've got full-time jobs or other commitments. It's good to follow up if you don't get a response, but wait a couple of days first.
 
+**Please do not address mentors as "sir" or "ma'am". We prefer to be addressed by our first names or usernames.**
```

Comment:
I would prefer using `Please refrain from addressing mentors as "sir" or "ma'am"`. Or `Please try to avoid addressing mentors as "sir" or "ma'am"`

Reply:
Fixed here: https://github.com/creativecommons/creativecommons.github.io-source/commit/8df38e362f1b3152d19229dcde65a605f41f212d